packages/replay-internal/src/types/performance.ts

```diff
@@ -114,7 +114,7 @@ export interface WebVitalData {
   /**
    * The layout shifts of a CLS metric
    */
-  attributions?: { value: number; sources?: number[] }[];
+  attributions?: { value: number; nodeIds?: number[] }[];
```

Comment:
Why did typescript not catch this?

Reply:
not sure, I think it should have tho since `getCumulativeLayoutShift` expects `ReplayPerformanceEntry<WebVitalData>` as the return type

Reply:
😬 we should follow-up after this PR is merged to see why the types aren't working as expected